subsys/bluetooth/host/hci_core.c

```diff
@@ -5372,6 +5375,26 @@ int bt_le_scan_stop(void)
 }
 #endif /* CONFIG_BT_OBSERVER */
 
+int bt_le_set_ch_map(u8_t ch_map[5])
+{
+	struct bt_hci_cp_le_set_host_chan_classif *cp;
+	struct net_buf *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_SET_HOST_CHAN_CLASSIF, sizeof(*cp));
```

Comment:
I realize almost all controllers will support this command in practice, however it'd still be good to formally check for it before attempting to do anything, since the core spec only mandates it for controllers supporting the Connection State. I.e. you should probably look at bt_dev.supported_commands first.